test/integration/journal.js

```diff
@@ -39,23 +39,46 @@ describe('test/integration/journal Journal Basic API', () => {
 });
 
 describe('test/integration/journal Journal Search API', () => {
-
-  const description = 'Sample voucher data one';
   const accountId = 187;
   const amount = 100;
   const DISTINCT_TRANSACTIONS = 22;
 
-  it(`GET /journal?description=${description} should match two records`, () => {
+  const includeNonPosted = 1;
+
+  // TODO(@jniles) - this flag seems backwards.  Shouldn't it be "includePostedValues"? or something of the sort?
```

Comment:
This does seem backward.  But it is deeply buried in the SQL queries, etc.   A bit convoluted.  But I did notice in journal.service.js:

``` { key : 'includeNonPosted', label : 'TRANSACTIONS.INCLUDE_POSTED_TRANSACTIONS_SHORT' },```

So it does seem like some cleanup on this is merited.